packages/p2p/src/components/temporarily-barred-hint/temporarily-barred-hint.tsx

```diff
@@ -5,7 +5,7 @@ import { useStores } from 'Stores';
 import { localize } from 'Components/i18next';
 import { getInlineTextSize } from 'Utils/responsive';
 
-const TemporarilyBarredHint = () => {
+const TemporarilyBarredHint = ({ is_my_ads_page = false }: { is_my_ads_page: boolean }) => {
```

Comment:
Isn't it better to check the pathname inside `TemporarilyBarredHint` component instead of passing a new prop?